doc/storage.md

```diff
@@ -397,6 +397,9 @@ sudo btrfs filesystem resize max /var/lib/lxd/storage-pools/<POOL>/
 ```bash
 losetup -l
 ```
+- For users of the snap version: if your btrfs volume is mounted inside the lxd snap's namespace `/run/snapd/ns/lxd.mnt`.
```

Comment:
I think I'd just update the instructions above to use `/var/snap/lxd/common/mntns/var/snap/lxd/common/lxd/` as that should always work for snap users.